Gemfile

```diff
@@ -0,0 +1,21 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.4.1'
+
+gem 'rails', '~> 5.2.0.rc2'
+gem 'pg', '>= 0.18', '< 2.0'
+gem 'puma', '~> 3.11'
+gem 'bootsnap', '>= 1.1.0', require: false
+
+group :development, :test do
+  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
```

Comment:
Byebug? Why not pry?

Reply:
was planning on adding this during the testing environment setup. But I will add it to this initial scaffold